src/Microsoft.ML.FastTree/BinFile/IniFileParserInterface.cs

```diff
@@ -254,8 +254,6 @@ public sealed class FeatureEvaluator
 
             public int Id { get; }
 
-            public MD5Hash ContentMD5Hash => MD5Hasher.Hash(Content);
```

Comment:
>ContentMD5Hash [](start = 27, length = 14)

Strange to have a fully unused field.  Looks like internal code does not use that either?